Project-Aurora/Project-Aurora/Settings/Bindables/Bindable.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Aurora.Utils;
+
+using Corale.Colore.Annotations;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Aurora.Settings.Bindables
+{
+    public class Bindable<T> : IBindable<T>, ISerializableBindable
```

Comment:
What sort of issues are these "Bindables" solving that aren't already solved by the existing VarRegistry. While the VarRegistry is by no stretch of the imagination a great example of config management, it has been doing the job for other devices for a long time. Whilst we are not against changing the VarRegistry by any means, having spoken to the other contributors, we currently do not see any advantage of using this over the existing method and we certainly don't want to have multiple config management systems doing effectively the same thing.

Reply:
When i wrote the hue code and i tried to actualy get the values back from the VarRegistry they were the default values and never returned the actual changed values and when i implemented this it fixed the issue.

Reply:
But why did you choose to implement a new system just for the Hue over attempting to fix the code that is already in place in many other places?

Reply:
As when i break pointed the code it wasn't even changed in the VarRegistry meaning it was something in WPF that most likely caused it not to be set.